### ISSUE TITLE
Chart with multiple dependencies with `requirements.yaml` not deploying all the pods

### ISSUE THREAD

**Issue by shettyh** [operator-framework/helm-app-operator-kit#36]
I have a chart with multiple dependent charts ( All are local charts only), structure is something like this

```
Chart
   - values.yaml
   - requirements.yaml
   - charts
         - depA
              - ...
          - depB
              - ...
          - etc
```

Operator deploys the chart properly and dont show any error. But only few pods are deployed.
Not sure why, `kubectl get events` also not showing any error.

If i try to to deploy the chart using `helm` cli client, it works just fine.

Any idea what needs to be done here ?

**Comment by alecmerdler** [comment]
We haven't looked at dependent Charts for this project, could you provide a simple example so we can try and reproduce? Thanks!

**Comment by shettyh** [comment]
@alecmerdler 
Please find the attached sample chart.

These are the things that are not working
- `services` are not created
- `pod` with PVC is not created (Looks like it)
- `values` file values are taken into consideration

[my-chart.zip](https://github.com/operator-framework/helm-app-operator-kit/files/2374717/my-chart.zip)


**Kubernetes Version**
```
Client Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.1", GitCommit:"b1b29978270dc22fecc592ac55d903350454310a", GitTreeState:"clean", BuildDate:"2018-07-17T18:53:20Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}
Server Version: version.Info{Major:"1", Minor:"11", GitVersion:"v1.11.1", GitCommit:"b1b29978270dc22fecc592ac55d903350454310a", GitTreeState:"clean", BuildDate:"2018-07-17T18:43:26Z", GoVersion:"go1.10.3", Compiler:"gc", Platform:"linux/amd64"}

```

**Comment by shettyh** [comment]
@alecmerdler 
I think it is not processing the multiple yaml definitions in a single file.

For example 

```

apiVersion: apps/v1
kind: Deployment                                          
metadata:
  name: rabbitmq
  namespace: {{ .Values.global.namespace }}
spec:
  replicas: {{ .Values.rabbitmq.replicas }}                                             
  minReadySeconds: 15

---

apiVersion: v1
kind: Service             
metadata:
  name: rabbitmq-service
  namespace: {{ .Values.global.namespace }}
spec:
  type: LoadBalancer       
  ports:
  - port: 5672              
    protocol: TCP          
    targetPort: 5672
  selector:                
    app: rabbitmq


```

Then only the first part of the file is considered. In this case only `Deployment` is created, and `Service` is not.

**Comment by shettyh** [comment]
@alecmerdler Requirements file is processed processed properly after #37 . But the above mentioned multi-resource in single file is still a issue

**Comment by alecmerdler** [comment]
Awesome, I suspected that was the issue. Looking forward to the PR!

**Comment by cpitman** [comment]
@shettyh Can you submit this pull request? I need the same feature for a template that generates 1..N resources!

**Comment by shettyh** [comment]
@cpitman I already raised PR #38 . It is not merged yet

**Comment by joelanford** [comment]
Merged PR #38. Thanks for submitting that fix @shettyh!
